pulsar/producer_partition.go

```diff
@@ -348,7 +348,7 @@ func (p *partitionProducer) internalSend(request *sendRequest) {
 		p.log.WithError(errMessageTooLarge).
 			WithField("size", len(payload)).
 			WithField("properties", msg.Properties).
-			Error()
+			Errorf("MaxMessageSize %d", int(p.cnx.GetMaxMessageSize()))
```

Comment:
Should we keep this in a structured form instead? 
```suggestion

			WithField("max-message-size", int(p.cnx.GetMaxMessageSize())).
			Error()
```